### DISCUSSION TITLE
Problème avec Edge

### DISCUSSION THREAD

**Discussion by jonathan-ati** [AmauriC/tarteaucitron.js#855] [General]
Bonjour, 

Je viens remonter un problème sur Edge, mais aléatoire et c'est le première fois que je rencontre ce problème. 
Je vais essayer d'être claire et précis. 

J'ai bien la licence tarteaucitron, je l'utilise pour gérer tous les sites avec l'interface. 

Quand, je me connecte sur le tarteaucitron fonctionne correctement. (pas de souci) 

Par contre, quand, je clique sur ma page "mon compte", le module apparait et disparait. 
Et vue que j'ai ma demande de consentement au milieu, le site reste bloqué sur l'écran blanc avec de l'opacité. 

Je ne sais pas quoi faire. 
Merci pour aide. 
Jo

**Comment by AmauriC** [comment]
👋 Possible d'avoir l'url du site pour que je regarde en détail?

**Reply by jonathan-ati** [reply]
Bonjour, 

Oui effectivement : 
https://www.universitepopulaire.fr/

**Reply by AmauriC** [reply]
Merci. Je viens de faire des tests (connecté ou non) et je n'arrive pas à reproduire le problème : /

Comme le mode auto est actif, vu les symptomes, ça pourrait être un service qui est chargé en ajax juste après le chargement de la page. Il n'y a rien de spécial en plus sur la page "mon compte"?

**Reply by jonathan-ati** [reply]
Merci pour la réponse, je vais voir avec nos développeurs.
